gunicorn_config.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+workers = 4
+errorlog = "/home/vcap/logs/gunicorn_error.log"
+bind = "0.0.0.0:{}".format(os.getenv("PORT"))
```

Comment:
Consider logging for the different states i.e, on start-up, shutdown etc

Maybe its one for the backlog?